js/utils/constants.js

```diff
@@ -585,6 +591,34 @@
       VOLUME_CHANGED: 'volumeChanged',
 
       /**
+       * A request to change the mute state has been made.
+       * The handler is called with the following arguments:
+       * <ul>
+       *   <li>The desired mute state of the video element.</li>
+       *   <li>The id of the video on which to mute (as defined by the module that controls it).
+       *        If null or undefined, all video elements volume will be changed</li>
+       * </ul>
+       *
+       *
```

Comment:
Do these need `@private` or `@public` as well?

Reply:
There is no consistency in the file about that, but I'll add @public since the skin makes use of them.